openmdao/utils/array_utils.py

```diff
@@ -355,6 +374,26 @@ def get_input_idx_split(full_idxs, inputs, outputs, use_full_cols, is_total):
         return [(inputs, full_idxs)]
 
 
+def convert_neg(arr, size):
```

Comment:
curious why this function exists since it's only used in the test... debugging?
also, why not get the size from the ndarray?